ytypes/common.go

```diff
@@ -768,7 +771,7 @@ func enumStringToIntValue(parent interface{}, fieldName, value string) (int64, e
 	}
 	ft := field.Type()
 	// leaf-list case
-	if ft.Kind() == reflect.Slice{
+	if ft.Kind() == reflect.Slice {
```

Comment:
With this change, it might be worth ensuring that `gofmt -w -s` doesn't make any changes to the existing code.

Reply:
I usually just run -w, this was a mis-format from a previous commit.